### PR TITLE
ExternalLinkRedirect_testFix

### PR DESCRIPTION
**Pull request by petrosinka**
Failing ‘externalLinks’ PortableInfobox test fixed
Link to Jira is added in ‘verifyReordering’ test

### CODE REVIEW

**Comment by petrosinka** [comment]
https://wikia-inc.atlassian.net/browse/WW-223

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfobox.java [hunk 1]
```diff
@@ -172,6 +172,12 @@ public String getDataImageName() {
     return imageTag.getAttribute("data-image-name");
   }
 
+  public PortableInfobox waitForUrlToContain(String expectedURL){
```
Comment (idradm) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfobox.java]: This seems like it should be a part of a page functionality, instead of an infobox.

File: src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java [hunk 2]
```diff
@@ -95,11 +95,11 @@ public void verifyExternalLinksRedirecting() {
...
         .getExternalLinkRedirectTitle(0);
 
-    String externalUrl = infobox
+    infobox
         .clickExternalLinkWithIndex(0)
-        .getUrlAfterPageIsLoaded();
+            .waitForUrlToContain(externalLinkName);
 
-    Assertion.assertEquals(externalLinkName.toLowerCase(), externalUrl.toLowerCase());
+    Assertion.assertEquals(externalLinkName, driver.getCurrentUrl());
```
Comment (idradm) [inline src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java]: Why did you remove URL normalization? Can different casing no longer happen?

File: src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java [hunk 3]
```diff
@@ -95,11 +95,11 @@ public void verifyExternalLinksRedirecting() {
         .open(PageContent.PORTABLE_INFOBOX_01)
         .getExternalLinkRedirectTitle(0);
 
-    String externalUrl = infobox
+    infobox
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java]: Looking above I think it makes sense to use the ArticleContent helper to pre-populate the page with what the test needs (infobox code)

File: src/test/java/com/wikia/webdriver/testcases/visualeditor/text/VisualEditorEditingTests.java [hunk 4]
```diff
@@ -104,7 +105,8 @@ public void VisualEditorEditing_002_delete() {
       groups = {"VisualEditorEditing", "VisualEditorEditing_003"},
       dependsOnGroups = "VisualEditorEditing_001"
   )
-  public void VisualEditorEditing_003_insertToExistingArticle() {
+  @RelatedIssue(issueID="WW-215)
```
Comment (sebastianmarzjan) [inline src/test/java/com/wikia/webdriver/testcases/visualeditor/text/VisualEditorEditingTests.java]: Syntax error - unclosed quote that causes the build to fail

**Comment by lukaszkonieczny** [comment]
This test works on production and preview for master branch
